src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
Please think of a better names for classes and class elements.
1. `Figure` IS NOT an `Area` -> `Figure` has an `Area` which can be provided. So how about `AreaProvider` or `AreaCalculator`?
2. `area()` does not match the method naming convention - how about `getArea()` or `calculateArea()` ?